src/models/core/SearchConfig.ts

```diff
@@ -31,7 +31,7 @@ export interface BaseSearchConfig {
   /**
    * Additional query params added on to every request.
    *
-   * @alpha
+   * @public
```

Comment:
can we just remove this tag now since it matches the parent interface's release tag?